operator-framework/src/main/java/com/github/containersolutions/operator/ControllerUtils.java

```diff
@@ -25,6 +25,12 @@
             doneableClassCache = new HashMap<>();
 
     static String getDefaultFinalizer(ResourceController controller) {
+        if(getAnnotation(controller).finalizerName().equals(DEFAULT_FINALIZER))
+        {
+            //[package+"."+classNameOfController]
+            return getAnnotation(controller).customResourceClass.getPackage().getName() + "." + getAnnotation(controller).customResourceClass.getSimpleName();
```

Comment:
This is more like the name of the customResource. Original idea was to use `controller.getClass()`